d1985r3.md

```diff
@@ -191,7 +191,7 @@ Here #2 fails late at #1 and #3 succeeds just as for the easy definition.
 
 ## Mechanism
 
-#### Additional possibilities offered by late ckecing
+#### Additional possibilities offered by late checking
```

Comment:
I think I have removed the notion of late checking. If it still there in main branch now that's a mistake. But you did have merge conflicts, maybe that's here.

Reply:
I have removed this entire section since.